### ISSUE TITLE
auro design tokens: add color tokens for different states

### ISSUE THREAD

**Issue by leeejune** [AlaskaAirlines/auro#254]
## Is your feature request related to a problem? Please describe.
There are no color tokens for backgrounds in a hover/active state.

## Describe the solution you'd like
Add color tokens to the design system.

## Additional context
We will need to add another name category for these new tokens. Ex: auro-color-ui-background-hover-on-dark

**Comment by leeejune** [comment]
@blackfalcon Are there any naming conventions that I should be aware of?

**Comment by blackfalcon** [comment]
@leeejune yes. Please review the names of all the existing tokens and there should be a logical placement of a new token. 

The naming is created based on the data structure of the token in the JSON. 

If you need help, please let me know.

**Comment by blackfalcon** [comment]
Also noticed that the settings.yml is really out of date. Will take this opportunity to get this up to speed.
